addons/google_recaptcha/__manifest__.py

```diff
@@ -16,6 +16,9 @@
         'web.assets_frontend': [
             'google_recaptcha/static/src/scss/recaptcha.scss',
             'google_recaptcha/static/src/js/recaptcha.js',
+        ],
+        #TODO in master set recaptcha.xml as a website.assets_wysiwyg asset.
+        'web.assets_common': [
```

Comment:
Why not making the right fix directly, what am I missing? :thinking:

Reply:
Maybe it's overthinking but I thought that maybe some custos were using this template.

Reply:
Ok, why not but...

- Using `assets_common` would still break those, it's not used in frontend screens anymore (all files of assets_common were moved inside the assets_frontend bundle).
- The TODO comment would need more explanation to explain why it is kept here.

So, good idea to think of people using the template it is currently in but then do this:
- Leave it in assets_frontend (frontend screen now only have this asset so moving it elsewhere would break custo as you suggested)
- Add a comment `# TODO remove me in master, it was kept here as part of a fix in stable but is unused in the current codebase` (something like that)
- Add the template in the asset where it needs to be added to be available in edit mode

:+1: